src/hooks/useSubscriptions.ts

```diff
@@ -41,18 +41,17 @@ export function useConfigSubscription(garden: Garden | null) {
 
     configSubscription.current = garden.onConfig(onConfigHandler)
 
-    return () => {
-      configSubscription.current = null
-    }
+    return () => configSubscription.current.unsubscribe()
   }, [garden, onConfigHandler])
 
   return config
 }
 
 export function useProposalsSubscription(filters: FiltersType) {
   const { config, connector } = useGardenState()
-  const [proposals, setProposals] = useState<any[]>([])
-  const proposalsSubscription = useRef(null)
+  const [proposals, setProposals] = useState<any>([])
```

Comment:
why did we change the type of the proposals here and generalised it to `any`?

Reply:
Done ✅